source/globalCommands.py

```diff
@@ -4304,18 +4307,19 @@ def script_reportLinkDestination(
 					link.destination,
 					# Translators: Informs the user that the window contains the destination of the
 					# link with given title
-					title=_("Destination of: {name}").format(
-						name=text,
-						closeButton=True,
-						copyButton=True,
```

Comment:
this looks like a strange logic error. is there a known bug associated with this?

Reply:
Are you referring to the following (in initial description of this PR)?
> ◦ Restored the Copy and Close buttons of the browseable message of NVDA+K that were removed by mistake in Report destination of link in Word / Outlook legacy #17292

If yes, the issue has not been logged anywhere else in GitHub; it's a regression of my own previous PR in the same dev cycle.